src/game/camera.c

```diff
@@ -828,6 +828,8 @@ s16 find_in_bounds_yaw_wdw_bob_thi(Vec3f pos, Vec3f origin, s16 yaw) {
             yaw = clamp_positions_and_find_yaw(pos, origin, 2458.f, -2458.f, 2458.f, -2458.f);
             break;
     }
+#else
+s16 find_in_bounds_yaw_wdw_bob_thi(UNUSED Vec3f pos, UNUSED Vec3f origin, UNUSED s16 yaw) {
```

Comment:
technically `yaw` is used tho this function is useless lol. maybe this whole function can be unincluded? idk where its used

Reply:
non blocking